.github/CODEOWNERS

```diff
@@ -492,4 +489,4 @@
 /package.json @vojtechjelinek @kevinlee12
 /yarn.lock @vojtechjelinek @kevinlee12
 /scripts/install_third_party_libs.py @vojtechjelinek @kevinlee12
-/.github/ @DubeySandeep
+/.github/ @seanlip
```

Comment:
This should be @kevinlee12 (dev workflow).

Reply:
Done!